gregoriotex/details.html

```diff
@@ -55,188 +55,492 @@
-
-<p>If you want to adjust the alignment by hand, you can change the second argument to a text with a height specification such as, for example, <code>\Large A</code> or <code>\tiny A</code>.  The height of that character string is used as a vertical offset.  If the second argument is, for example, 9 pt. in height, then the first argument will be displayed with the baseline of its text 9 pt. below the top line of the staff.   Therefore, a taller second argument makes the first argument display lower, and a less tall second argument makes the first argument display higher.</p>
-
-<p>To add a note about source of the text, call the macro <code>\commentary{}</code> which takes a text argument.   It will be displayed, right-aligned, just above the score.  This macro (and <code>\gresetfirstlineaboveinitial</code> must be called immediately before the <code>\includescore</code> without an intervening blank line.</p>
-
-
-
-<h3>Modifying the vertical spacing</h3>
-
-<p>Gregorio adjusts vertical spacing automatically whenever there is an additional line, with the macro <code>\grenewlinewithspace{}{}</code>, described below.  Still, the vertical spacing may not be ideal for your purposes, so you can adjust spacing in several ways:</p>
-
-<ul>
-<li><code>\spaceabovelines</code> for the space above the staff</li>
-<li><code>\spacelinestext</code> for the space between the baseline of the text and the bottom line of the staff</li>
-<li><code>\spacebeneathtext</code> for the space below the text</li>
-<li><code>\translationheight</code> for space dedicated to a translation of the text, if there is one</li>
-</ul>
-
-<p>To change other spaces, copy the file <code>gsp-default.tex</code>, rename it in <code>gsp-myspaces.tex</code>, change the values of the spaces and call <code>\GreLoadSpaceConf{myspaces}</code>.</p>
-
-<p>It is also possible to change TeX algorithm for spacing. For instance try to put this in your code before any score inclusion:</p>
-
-<div class="commandline">
-<code>\tolerance=9999</code><br />
-<code>\pretolerance=500</code>
-</div>
-
-<p>It should give a better result, but also more spacing and thus more lines.</p>
+<p>To add indications above the initial letter, you can use the macro <code>\greannotation{}</code>.  The argument is the text you wish to set there.  To get multiple lines, simply call the command multiple times.  Each call will add a new line to the bottom containing the new text.</p>
+
+<p>By default new lines are alinged center with the existing content.  If you want to adjust the horizontal alignment by hand, add the optional argument <code>[r]</code> or <code>[l]</code> for right and left alignment respectively: e.g. <code>\greannotation[r]{right aligned second line}</code></p>
```

Comment:
I think `If you want to adjust the horizontal alignment by hand` should be changed to `If you want a different horizontal alignment`, since you can only select an alignment and not make finer adjustments.